src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int top;
+    private int bottom;
+    private int high;
+
+    public IsoscelesTrapezoid(int top, int bottom, int high, Color color) {
+        super(color);
+        this.high = high;
+        this.bottom = bottom;
+        this.top = top;
+    }
+
+    @Override
+    public double getArea() {
+        return ((top + bottom) / 2) * high;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: IsoscelesTrapezoid, Area: " + getArea()
+                + " sq. units, Top: " + top + " Bottom: " + bottom
+                + " High: " + high 
```

Comment:
Replace 'High' with 'Height' in the output string to match the recommended variable name change. 🚀